### PR TITLE
[Merged by Bors] - Adds feedback and debug info to 'smart-module create'

### PR DESCRIPTION
**Pull request by JacobBoling**
Fixes issue #2007

### CODE REVIEW

File: crates/fluvio-cli/src/smartmodule/create.rs [hunk 1]
```diff
@@ -34,8 +35,12 @@ impl CreateSmartModuleOpt {
             wasm: SmartModuleWasm::from_binary_payload(buffer),
             ..Default::default()
         };
+        
+        debug!("creating smart-module: {}", self.name);
```
Comment (sehz) [inline crates/fluvio-cli/src/smartmodule/create.rs:39]: can you use structure logging? https://docs.rs/tracing/0.1.35/tracing/macro.debug.html

**Review by sehz — CHANGES_REQUESTED** [review]

File: crates/fluvio-cli/src/smartmodule/create.rs (hunk 1 above)
Comment (JacobBoling) [reply crates/fluvio-cli/src/smartmodule/create.rs:39]: Did you put in lines 38 and 39? Sorry, I thought I did that.

Comment (sehz) [reply crates/fluvio-cli/src/smartmodule/create.rs:39]: something like this:
```
debug!(name = self.name, "creating smart-module");
```

File: crates/fluvio-cli/src/smartmodule/create.rs [hunk 2]
```diff
@@ -34,8 +35,12 @@ impl CreateSmartModuleOpt {
...
         let admin = fluvio.admin().await;
         admin.create(self.name.to_string(), false, spec).await?;
+        println!("smart-module \"{}\" has been created.", self.name);
```
Comment (sehz) [inline crates/fluvio-cli/src/smartmodule/create.rs:42]: why blackslash?

Comment (JacobBoling) [reply crates/fluvio-cli/src/smartmodule/create.rs:42]: That is how the message is structured in topic/create

`println!("topic \"{}\" created", name);`

Comment (sehz) [reply crates/fluvio-cli/src/smartmodule/create.rs:42]: k

File: crates/fluvio-cli/src/smartmodule/create.rs (hunk 1 above)
Comment (JacobBoling) [reply crates/fluvio-cli/src/smartmodule/create.rs]: Ok I changed the line to that. Also letting you know how it is done in topic/create:

`debug!("creating topic: {} spec: {:#?}", name, topic_spec);`

Comment (sehz) [reply crates/fluvio-cli/src/smartmodule/create.rs]: Yup.  There's lots of code using legacy format.

**Review by sehz — APPROVED** [review]
LGTM

**Comment by sehz** [comment]
`cargo fmt` check failure

**Pull request closed without merging**